lib/language_pack/ruby.rb

```diff
@@ -29,7 +29,7 @@ def self.use?
   end
 
   def self.bundler
-    @bundler ||= LanguagePack::Helpers::BundlerWrapper.new.install
+    @@bundler ||= LanguagePack::Helpers::BundlerWrapper.new.install
```

Comment:
Yeah, I don't exactly understand why, but I think it is not working as intended with just `@bundler`. I put a print statement in the BundleWrapper initialize method and ran the buildpack and saw it be initialized multiple times (once for each ruby subclass I think). Changing to `@@bundler` caused BundlerWrapper to only be initialized once.